pythran/spec.py

```diff
@@ -151,8 +153,13 @@ def p_opt_types(self, p):
         p[0] = p[1] if len(p) == 2 else []
 
     def p_types(self, p):
-        '''types : type
-                 | type COMMA types'''
+        '''types : multitype
+                 | multitype COMMA types'''
```

Comment:
It looks like this commit supersedes the optional stuff, why should we propose two ways to achieve the same goal?